prijateli_tree/app/routers/games.py

```diff
@@ -383,7 +391,7 @@ def score_to_denirs(
     return {"reward": f"You have made {denirs} denirs!"}
 
 
-@router.post("/{game_id}/player/{player_id}/integrated")
+@router.get("/{game_id}/player/{player_id}/integrated")
```

Comment:
The game shouldn't need to know what type of network is being used, that should be dealt with by the endpoint logic. This should just be `/{game_id}/player/{player_id}` and you should route it to an integrated game function.